app/javascript/controllers/filter_controller.js

```diff
@@ -9,6 +9,10 @@ export default class extends Controller {
     
         const params = new URLSearchParams(formData);
         const newUrl = `${form.action}?${params.toString()}`;
+
+        const defaultSkillForms = document.querySelectorAll('#default-skills form');
+        defaultSkillForms.forEach(f => f.action = newUrl)
```

Comment:
This is not understandable when reading the code, maybe a small comment or simplify.